src/robots/gazebo/uav_plugin.cc

```diff
@@ -345,13 +351,13 @@ GazeboQuadCopterPlugin::ApplyMotorForces(const double _dt)
 {
     MotorMixing(_dt);
     // update velocity PID for rotors and apply force to joint
-    for (size_t i = 0; i < m_Rotors.size(); ++i) {
-        const double velTarget = m_Rotors[i].m_Multiplier *
-                           m_Rotors[i].m_Cmd /
-                           m_Rotors[i].m_RotorVelocitySlowdownSim;
-        const double vel = m_Rotors[i].m_Joint->GetVelocity(0);
+    for (auto &rotor : m_Rotors) {
```

Comment:
oh wow - did clang-tidy spot that this loop could be achieved without messing about with indices? 😮

Reply:
Yes! And it even had a go at writing a patch for it. I did clean it up a bit, but it's still nice!